core/tee/tee_svc.c

```diff
@@ -256,6 +281,371 @@ TEE_Result syscall_get_property(unsigned long prop, void *buf, size_t blen)
 	}
 }
 
+static TEE_Result get_prop_tee_dev_id(struct tee_ta_session *sess,
```

Comment:
I made it like this so that removing the "obsolete" will easy. In order to remove duplicated code, I will use, in the obsolete version, the new functions `get_prop_tee_dev_id()`...